client/blocks/jetpack-benefits/site-backups.tsx

```diff
@@ -0,0 +1,169 @@
+	} );
+
+	const dispatch = useDispatch();
+	const refreshBackupProgress = useCallback( () => dispatch( requestRewindBackups( siteId ) ), [
+		dispatch,
+		siteId,
+	] );
+
+	const loadingBackups = backups === null; // getRewindBackups returns null if there are no backups for the siteId
+
+	const backupCurrentlyInProgress = useSelector( ( state ) => {
+		return getInProgressBackupForSite( state, siteId );
+	} );
+
+	const getBackupTimeAgo = ( backup: { last_updated: string } ) => {
+		return moment.utc( backup.last_updated ).fromNow();
+	};
+
+	const getPostCountForBackup = ( backup ) => {
+		const stats = backup.stats;
+		return (
+			stats.tables[ stats.prefix + 'posts' ].post_published ??
+			stats.tables[ stats.prefix + 'posts' ].published
+		);
+	};
+
+	const getStatForBackup = ( backup, statName: string ) => {
+		const stats = backup.stats;
+		return stats[ statName ] ? stats[ statName ].count : 0;
+	};
+
+	// if still loading, show a loading state
+	if ( loadingBackups ) {
+		return (
+			<React.Fragment>
+				<JetpackBenefitsCard
+					jestMarker="loading-backups" // for test suite
+					headline={ translate( 'Site Backups' ) }
+					description={ translate( 'Loading backup data' ) }
+					stat={ translate( 'Placeholder' ) }
+					placeholder={ true }
+				/>
+			</React.Fragment>
+		);
+	}
+
+	// If a backup is in progress,
+	// start tracking and showing progress
+	if ( backupCurrentlyInProgress ) {
+		return (
+			<>
+				<Interval onTick={ refreshBackupProgress } period={ EVERY_SECOND } />
+				<JetpackBenefitsCard
+					headline={ translate( 'Site Backups' ) }
+					description={
+						<ProgressBar
+							value={ backupCurrentlyInProgress.percent }
+							total={ 100 }
+							color="#069E08"
+						/>
+					}
+					stat={ translate( 'In Progress' ) }
+				/>
+			</>
+		);
+	}
+
+	// now that backups are loaded and any in progress are complete, get the most recent one
+	const mostRecentBackup = backups.length > 0 ? backups[ 0 ] : null;
+
+	// no backups taken yet
+	if ( ! mostRecentBackup ) {
+		return (
+			<JetpackBenefitsCard
+				jestMarker="no-backups" // for test suite
+				headline={ translate( 'Site Backups' ) }
+				description={ translate( 'Jetpack will back up your site soon.' ) }
+			/>
+		);
+	}
+
+	// most recent backup was not successful
+	if ( mostRecentBackup.status !== 'finished' && mostRecentBackup.status !== 'started' ) {
+		return (
+			<JetpackBenefitsCard
+				jestMarker="recent-backup-error" // for test suite
+				headline={ translate( 'Site Backups' ) }
+				description={ translate(
+					"Jetpack's last attempt to back up your site was not successful. Please contact Jetpack support."
+				) }
+				stat="Error"
+			/>
+		);
+	}
+
+	const lastBackupTimeAgo = getBackupTimeAgo( mostRecentBackup );
+	if ( isStandalone ) {
+		return (
+			<JetpackBenefitsStandaloneCard
+				title={ translate( 'Site Backups' ) }
+				summary={ {
+					title: translate( 'Last Backup' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Last backup was on:' )` *ES Score: 7*